resources/less/main.less

```diff
@@ -1,7 +1,6 @@
 // Bootstrap UI demo
 
-@import "bootstrap.ui/src/less/bootstrap-ui";
+@import "../../node_modules/bootstrap-ui/src/less/bootstrap-ui";
 
 // Settings
-@external-components-path: '../bower_components/';
-@brand-primary: @gray-dark;
+@external-components-path: '../../../../node_modules/';
```

Comment:
I think I got it. It works with both of these:

1. `'node_modules/'` which is relative to `Gruntfile.js`
2. `'../../../'` which is the same as `'../../../../node_modules/'`, both of which are relative to `bootstrap-ui/src/less/bootstrap-ui.less` where the variable is called

I would prefer using `'node_modules/'` as it seems more understandable to me.

Anyway, we got it [covered in the docs](http://docs.bootstrap-ui.com/#customizing).